docs/aws/audit/cloudwatchmonitoring/rules/iam_policy_changes_alarm.mdx

```diff
@@ -24,101 +24,71 @@ SOC2, HIPAA, ISO27001, AWSWAF, HITRUST, CISAWS, CBP, NISTCSF, PCIDSS
 
 #### Using Console
 
-The IAM Policy Changes Alarm is triggered when there is a change in the IAM policies that are attached to an IAM user, group or role. This alarm is important to monitor as it can help detect unauthorized changes to your IAM policies that could potentially lead to security breaches. Here are the steps to remediate this issue:
+1. **Sign in to the AWS Management Console**: Go to the AWS Management Console at https://console.aws.amazon.com/.
```

Comment:
console steps show how to find if it exists or not. but, not a remediation step to enable the alarm.